### DISCUSSION TITLE
About the function "CO2_biogen()"

### DISCUSSION THREAD

**Discussion by XuanhongZhou** [UMEP-dev/UMEP#564] [Q&A]
hi~ 
I am a beginner of SUEWS. In my work, i only need to use the function "CO2_biogen()" in subroutine co2_module. However, i found some problems when i was looking for the data.
1. Here are some variables that i don't know how to find them:  "gfunc, gfunc, temp_c, t2". The document lacks the descriptions of them.
2. The array "SnowFrac", it contains the fraction of snow on different surfaces, i found its description on the website "https://suews.readthedocs.io/en/latest/input_files/Initial_Conditions/Snow_related_parameters.html", however, i found nothing on the configuration part. And here is the same problem with "FAI_id".
I hope fellow scholars and colleagues can provide me with answers, and I am truly grateful.

**Comment by biglimp** [comment]
@sunt05

**Comment by sunt05** [comment]
Hello @XuanhongZhou, could you please specify the environment in which you're utilising SUEWS? Is it through UMEP or supy? If uncertain, feel free to share a screenshot. thanks.
